### PR TITLE
Add Chinese Translation (2023.1.23) 

### PR DESCRIPTION
**Pull request by SleepyAsh0191**
Happy Lantern Rite!

It's my first pr in this repo.
I added the Chinese words in these file. And add some missing translation in quests.
I also found the 3.4 Sub Event `Paper Theater` is missing, so I added it in events file.

Thanks for your review.

### CODE REVIEW

**Comment by xicri** [comment]
@SleepyAsh0191 Thanks for the contribution! Your contribution helps us a lot!

@Bill-Haku I've reviewed this PR, but can you review this PR too?

**Review by xicri — DISMISSED** [review]

**Comment by Bill-Haku** [comment]
No problem. I will do it.

**Comment by SleepyAsh0191** [comment]
sry wrong click

File: dataset/dictionary/characters.json5 [hunk 1]
```diff
@@ -1453,6 +1458,9 @@
     en: "Yae Miko",
     ja: "八重神子",
     zhCN: "八重神子",
+    variants: {
+      zhCN: [ "八重寄子" ],
+    },
```
Comment (Bill-Haku) [inline dataset/dictionary/characters.json5:1463]:
```suggestion
```
I don't think we need to add such an insulating name in the variants.

**Review by Bill-Haku — CHANGES_REQUESTED** [review]

**Comment by xicri** [comment]
@Bill-Haku Thanks for the review!

@SleepyAsh0191 Thanks for updating code! To merge this PR, can you sign with the Contributor License Agreement?

**Comment by SleepyAsh0191** [comment]
yes

**Review by xicri — APPROVED** [review]

**Comment by xicri** [comment]
@SleepyAsh0191 Thanks, I merged it. It will be deployed in a few minutes.

**Pull request merged**